src/templates/artworks.js

```diff
@@ -6,69 +6,96 @@ import { rhythm } from "../utils/typography"
 
 import Layout from "../components/Layout"
 import Content, { HTMLContent } from "../components/Content"
+import Features from '../components/Features'
 
-const ArtworkTemplate = ({ title, content, contentComponent }) => {
+const ArtworkTemplate = ({ title, content, contentComponent, intro, heading, }) => {
   const PageContent = contentComponent || Content
   return (
       <div className="container content">
        <h1>{title}</h1>
+       {heading}
+       <Features gridItems={intro.blurbs} />
       <PageContent className="content" content={content} />
       </div>
 )
 }
 
 ArtworkTemplate.propTypes = {
+  image: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
+  heading: PropTypes.string,
   title: PropTypes.string.isRequired,
   content: PropTypes.string,
   contentComponent: PropTypes.func,
+  intro: PropTypes.shape({
+    blurbs: PropTypes.array,
+  }),
 }
 
-class ArtworksPage extends React.Component {
-
-  render() {
-    var dataMarkdown = [];
-    if (this.props.data !== null) {
-      dataMarkdown = this.props.data.markdownRemark
-    }
+const ArtworksPage = ({ data, props }) => {
+  const { frontmatter } = data.markdownRemark;
+  //const location = props.location;
+  //console.log(location);
     return (
-      <Layout className="container" data={this.props.data} location={this.props.location}>
+      <Layout className="container" data={data} >
```

Comment:
With these mods i can not pass the location object as before, i have to find another approach.